client/src/components/card/mobile-card-component.tsx

```diff
@@ -12,81 +12,84 @@ interface CardReactProps
 	> {
 	cards: Array<LocalCardInstance>
 	tooltipAboveModal?: boolean
-	onClick?: () => void
+	onSubtractionClick?: () => void
 	onAdditionClick?: () => void
 	small: boolean
 }
 
 const MobileCardComponent = (props: CardReactProps) => {
-	const {onClick, onAdditionClick, cards, small} = props
+	const {onSubtractionClick, onAdditionClick, cards, small} = props
 
 	return (
 		<Tooltip
 			tooltip={<CardInstanceTooltip card={props.cards[0].props} />}
 			showAboveModal={props.tooltipAboveModal}
 		>
 			<div className={css.MobileCardComponentContainer}>
-				<button onTouchStart={onClick} className={css.mainButton}>
-					<div
-						className={classNames(css.MobileCardComponent, small && css.small)}
-					>
-						{cards[0].props.category === 'hermit' && (
-							<div>
-								<img
-									className={css.headInList}
-									src={`/images/hermits-emoji/${cards[0].props.id.split('_')[0]}.png`}
-								/>
-								{small && (
-									<div
-										className={classNames(
-											css.rarityStar,
-											css[cards[0].props.rarity],
-										)}
-									></div>
-								)}
-							</div>
-						)}
-						{(cards[0].props.category === 'attach' ||
-							cards[0].props.category === 'single_use') && (
+				<div
+					className={classNames(css.MobileCardComponent, small && css.small)}
+				>
+					{cards[0].props.category === 'hermit' && (
+						<div>
 							<img
 								className={css.headInList}
-								src={`/images/effects/${cards[0].props.id}.png`}
+								src={`/images/hermits-emoji/${cards[0].props.id.split('_')[0]}.png`}
 							/>
-						)}
-						{cards[0].props.category === 'item' && (
-							<div>
-								<img
-									className={css.headInList}
-									src={`/images/types/type-${cards[0].props.id.split('_')[1]}.png`}
-								/>
-								{small && (
-									<div
-										className={classNames(
-											css.rarityStar,
-											css[cards[0].props.rarity],
-										)}
-									></div>
-								)}
-							</div>
-						)}
+							{small && (
+								<div
+									className={classNames(
+										css.rarityStar,
+										css[cards[0].props.rarity],
+									)}
+								></div>
+							)}
+						</div>
+					)}
+					{(cards[0].props.category === 'attach' ||
+						cards[0].props.category === 'single_use') && (
+						<img
+							className={css.headInList}
+							src={`/images/effects/${cards[0].props.id}.png`}
+						/>
+					)}
+					{cards[0].props.category === 'item' && (
+						<div>
+							<img
+								className={css.headInList}
+								src={`/images/types/type-${cards[0].props.id.split('_')[1]}.png`}
```

Comment:
Theres a function called `getTypeImage` or something I think you can use here. We use this image in a lot of places so it needs to be consistent.